projects/telemetry/inc/nmea.h

```diff
@@ -0,0 +1,25 @@
+// # The NMEA message format
```

Comment:
Is this module all that the NMEA format is associated with? If there's more to the format, I might recommend remaining this to `nmea_checksum` or something similar.

Reply:
Ok, done